### ISSUE TITLE
Build task: "tsc build" throws "Cannot read property 'forEach' of undefined"

### ISSUE THREAD

**Issue by miseifert** [Microsoft/vscode#64324]
```
Version: 1.30.0-insider (user setup)
Commit: fa87e67ebfd6b8014ba697ddbc6c2d6c0e2cb970
Date: 2018-12-04T09:34:30.209Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
OS: Windows_NT x64 10.0.17134
```

Steps to Reproduce:

1. Press <kbd>Ctrl </kbd> + <kbd>Shift</kbd> + <kbd>B</kbd>
2. Choose "tsc build ... tsconfig.json" or "tsc watch ..."

A shortened screenshot that shows the chosen option and error message:

![unbenannt](https://user-images.githubusercontent.com/36030114/49441404-a6deba80-f7c6-11e8-8d93-9a0a10363085.png)

Also happens with extensions disabled.

**Comment by dbaeumer** [comment]
Not sure if this is us or @mjbvz

**Comment by alexr00** [comment]
This is an issue with tasks.

**Comment by alexr00** [comment]
Fixed by 939d390bc31a074

**Comment by miseifert** [comment]
Thank you very much!
